config/blockchain.js

```diff
@@ -1,149 +1,31 @@
-if (process.env.WALLET_PASSWORD == undefined) {
-  throw Error('Env variable WALLET_PASSWORD not defined!')
+if (!process.env.WALLET_MNEMONIC) {
+  throw Error('Env variable WALLET_MNEMONIC not defined!')
 }
 
 module.exports = {
-  // applies to all environments
   default: {
     enabled: true,
-    rpcHost: 'localhost', // HTTP-RPC server listening interface (default: "localhost")
-    rpcPort: 8545, // HTTP-RPC server listening port (default: 8545)
-    rpcCorsDomain: {
-      // Domains from which to accept cross origin requests (browser enforced). This can also be a comma separated list
-      auto: true, // When "auto" is true, Embark will automatically set the cors to the address of the webserver
-      additionalCors: [], // Additional CORS domains to add to the list. If "auto" is false, only those will be added
-    },
-    wsRPC: true, // Enable the WS-RPC server
-    wsOrigins: {
-      // Same thing as "rpcCorsDomain", but for WS origins
-      auto: true,
-      additionalCors: [],
-    },
-    wsHost: 'localhost', // WS-RPC server listening interface (default: "localhost")
-    wsPort: 8546, // WS-RPC server listening port (default: 8546)
-
-    // Accounts to use as node accounts
-    // The order here corresponds to the order of `web3.eth.getAccounts`, so the first one is the `defaultAccount`
-    // accounts: [
-    // {
-    //   nodeAccounts: true, // Accounts use for the node
-    //   numAddresses: '1', // Number of addresses/accounts (defaults to 1)
-    //   password: 'config/development/password', // Password file for the accounts
-    // },
-    // Below are additional accounts that will count as `nodeAccounts` in the `deployment` section of your contract config
-    // Those will not be unlocked in the node itself
-    // {
-    //   privateKeyFile: 'path/to/file', // Either a keystore or a list of keys, separated by , or ;
-    //   password: 'passwordForTheKeystore', // Needed to decrypt the keystore file
-    // },
-    // {
-    //   mnemonic: '12 word mnemonic',
-    //   addressIndex: '0', // Optional. The index to start getting the address
-    //   numAddresses: '1', // Optional. The number of addresses to get
-    //   hdpath: "m/44'/60'/0'/0/", // Optional. HD derivation path
-    // },
-    // ],
-  },
-
-  // default environment, merges with the settings in default
-  // assumed to be the intended environment by `embark run` and `embark blockchain`
-  development: {
-    networkType: 'testnet',
-    syncMode: 'light',
+    client: 'geth',
```

Comment:
Also, this change means I need geth installed on my local machine, correct? I was happier when this wasn't the case and I could develop on localhost, but still connect to a Ropsten instance of the contracts so that it was more real than just having everything local...

Reply:
I removed the line, but to be fair, it changes nothing. As you can see on the lines below, `development` and the other envs all point to Inufra, so even though Embark defaults to Geth as a client, Geth will never be started because Embark will connect to those endpoints first.